deploy/operator/mirror_utils.sh

```diff
@@ -46,10 +46,17 @@ function mirror_package() {
         --registry-config="${authfile}" \
         --to-manifests="${manifests_dir}"
 
-  echo "Applyed image-content-source-policy:"
-  cat "${manifests_dir}/imageContentSourcePolicy.yaml"
-
-  oc apply -f "${manifests_dir}/imageContentSourcePolicy.yaml"
+  # Starting with version 4.14 of OpenShift the ImageContentSourcePolicy object is deprecated, and
+  # replaced by ImageDigestMirrorSet. The 'oc adm catalog mirror' command generates both, so we
+  # check and use the new one if possible.
+  if [ -f "${manifests_dir}/imageDigestMirrorSet.yaml" ]; then
+    mirrors_config_file="${manifests_dir}/imageDigestMirrorSet.yaml"
+  else
+    mirrors_config_file="${manifests_dir}/imageContentSourcePolicy.yaml"
+  fi
+  echo "Applyed mirrors configuration:"
```

Comment:
This should read "Applied mirrors configuration:"

Reply:
Done, thanks @paul-maidment .